src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball {
+    private String colour;
```

Comment:
You have a method `getRandomColor()` (named without `u`). Make everywhere names without `u`.